ZConfig/cmdline.py

```diff
@@ -142,7 +174,10 @@ def _normalize_case(self, string):
         return string.lower()
 
 
-class MatcherMixin:
+class MatcherMixin: # pylint:disable=old-style-class
+    # Can't extend object without getting __init__ errors due to
```

Comment:
Can't this be handled by defining an appropriate __init__ instead of telling pylint to ignore this?